articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+
+  1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+    </body>
+  </html>
+  ~~~
+
+  2. **Page-Level styles:** Page-level styles are defined at the header area of the HTML file. Unlike local styles, you can customize all the similar tags at once thereby saving time and maintaining uniformity.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+      <meta charset="utf-8">
+      <style type="text/css">
+         body{
+           color: yellow;
+           background-color: red;
+         }
+         p{
+           color: red;
+           background-color: yellow;
+         }
+      </style>
+    </head>
+    <body>
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+  ~~~
+
+  3. **External Styles** In this case, the styles used for the webpage are located in a completely different file. They are linked to the HTML file depending on which CSS file (Style) best fits the webpage. When developing a website with multiple pages this styling usually comes in handy.
+
+  ~~~html
+  <!DOCTYPE html>
+  <html>
+    <head>
+        <meta charset="utf-8">
+        <title>External Styles</title>
+        <link rel="stylesheet" type="text/css" href="css/myStyle.css"/>
+    </head>
+    <body>
+      <h1>Heading</h1>
+        <p>This is an example of External CSS</p>
+    </body>
+  </html>
+  ~~~
+  ~~~CSS
+  /*This is the CSS file called myStyle.css*/
+  body{
+    background-color: black;
+    color: white;
+  }
+  p{
+    color: purple;
+  }
+  ~~~
+
+### When to use each Style
+With these three options for styling in CSS, it becomes rather difficult to identify the most appropriate when building a webpage.
+
+  * _Local Style_ is not the first choice that comes to mind when dealing with CSS. At some point, it becomes too much of a burden to keep styling each element individually. By the time you come to the end of your project, the entire file is a mess. Even the smallest changes will require you to redo at least **70%** of the CSS. Which in the end prolongs the time needed to complete the project.
+
+  However, This does not mean that the style should not be used. It usually works well when you're trying to test out a new style on an element making it easier to determine if the style works well with the design or not. But always keep in mind that local styles have a higher priority over page-level and external styles. Meaning they will override any style property defined in page-level and external
```

Comment:
Overarching comment for this entire section:
It might be better to do 2 bullet points for each section to cut through the fluff:

- **Pros**
- **Cons**

For inline vs external, you might think of implications like page rendering and download time when you have to wait for an external stylesheet to load, or page size when you have so many individual elements styled that it takes a long time to load every one of them. 

Clean code in separate files might be a pro, easily testing inline might be a pro, etc.